.github/workflows/ci.yml

```
@@ -120,38 +120,32 @@ jobs:
         working-directory: integration-tests
         run: cargo test --release --locked --features "${{ matrix.features }}"
 
-  api-tests:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-
-      - uses: "./.github/actions/init"
-
-      - name: Run tests
-        working-directory: pop-api
-        run: cargo test --release --locked --all-features
-
-  api-integration-tests-devnet:
+  pop-api-integration-tests:
+    needs: lint
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        features:
+          - "devnet"
+          - "testnet"
     steps:
       - uses: actions/checkout@v4
 
       - uses: "./.github/actions/init"
 
-      - name: Run integration tests
-        working-directory: pop-api/integration-tests
-        run: cargo test --release --locked --features devnet
+      - name: Run pop api integration tests
```

```suggestion
      - name: Run API integration tests
```
Feel free to ignore.

---

changed in https://github.com/r0gue-io/pop-node/pull/434/commits/b4ba54746511858e051486698eaa26dc07a106f1